config/routes.rb

```diff
@@ -1,8 +1,14 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root to: 'home#index'
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :availabilities do
```

Comment:
If you don't need all CRUD from `availabilities`, filter with `only` or `except`